src/openvtb/components/taken/api/tests/test_betaaltaak.py

```diff
@@ -0,0 +1,641 @@
+                "reason": "Dit veld is vereist.",
+            },
+        )
+        self.assertEqual(ExterneTaak.objects.all().count(), 0)
+
+        # empty details values
+        data = {
+            "titel": "test",
+            "handelingsPerspectief": "test",
+            "details": {},
+        }
+        response = self.client.post(self.list_url, data)
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+        self.assertEqual(response.data["code"], "invalid")
+        self.assertEqual(response.data["title"], "Invalid input.")
+        self.assertEqual(len(response.data["invalid_params"]), 3)
+        self.assertEqual(
+            get_validation_errors(response, "details.bedrag"),
+            {
+                "name": "details.bedrag",
+                "code": "required",
+                "reason": "Dit veld is vereist.",
+            },
+        )
+        self.assertEqual(
+            get_validation_errors(response, "details.transactieomschrijving"),
+            {
+                "name": "details.transactieomschrijving",
+                "code": "required",
+                "reason": "Dit veld is vereist.",
+            },
+        )
+        self.assertEqual(
+            get_validation_errors(response, "details.doelrekening"),
+            {
+                "name": "details.doelrekening",
+                "code": "required",
+                "reason": "Dit veld is vereist.",
+            },
+        )
+        # details.doelrekening empty values
+        data = {
+            "titel": "test",
+            "handelingsPerspectief": "test",
+            "details": {
+                "bedrag": "12",
+                "transactieomschrijving": "12",
+                "doelrekening": {},
+            },
+        }
+        response = self.client.post(self.list_url, data)
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+        self.assertEqual(response.data["code"], "invalid")
+        self.assertEqual(response.data["title"], "Invalid input.")
+        self.assertEqual(len(response.data["invalid_params"]), 2)
+        self.assertEqual(
+            get_validation_errors(response, "details.doelrekening.naam"),
+            {
+                "name": "details.doelrekening.naam",
+                "code": "required",
+                "reason": "Dit veld is vereist.",
+            },
+        )
+        self.assertEqual(
+            get_validation_errors(response, "details.doelrekening.iban"),
+            {
+                "name": "details.doelrekening.iban",
+                "code": "required",
+                "reason": "Dit veld is vereist.",
+            },
+        )
+
+    def test_invalid_create_pass_soort_taak(self):
+        self.assertEqual(ExterneTaak.objects.all().count(), 0)
+        # wrong soort_taak
+        data = {
+            "titel": "test",
+            "handelingsPerspectief": "test",
+            "taak_soort": SoortTaak.FORMULIERTAAK.value,
```

Comment:
It should be camelcased, right?
```suggestion
            "taakSoort": SoortTaak.FORMULIERTAAK.value,
```